deliver/spec/commands_generator_spec.rb

```diff
@@ -15,7 +15,7 @@ def expect_runner_new_with(expected_options)
   end
 
   describe ":run option handling" do
-    it "can use the username short flag from tool options", now: true do
+    it "can use the username short flag from tool options" true do
```

Comment:
The `true` should probably be removed also, right?

Reply:
It was in another commit that @taquitos added to the main PR.

Reply:
Thanks for confirming @ohayon 👍